src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.js

```diff
@@ -156,10 +156,9 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
 
     const timePeriodOptions = useCallback(
         () =>
-            _.map(statusType, (item, index) => (
-                <BaseListItem
+            _.map(statusType, (item) => (
+                <RadioListItem
                     item={item}
-                    key={`${index}+${item.value}`}
```

Comment:
While testing the app, I'm seeing this error:
May I know you removed the `key` props? @luacmartins 

https://github.com/Expensify/App/assets/16502320/7a24f2c6-b4ee-4bb5-9a36-42079e61e42f

Reply:
Hmm we removed it because RadioListItem adds a key [here](https://github.com/Expensify/App/blob/581f72058e8710c16a2105328654ce4fa157ae61/src/components/SelectionList/RadioListItem.tsx#L36), but maybe we need that at the higher level.

Reply:
Haha yeah 😅